axiom/nr_txn.c

```diff
@@ -3276,6 +3276,18 @@ bool nr_txn_log_forwarding_enabled(nrtxn_t* txn) {
   return true;
 }
 
+bool nr_txn_log_forwarding_context_data_enabled(nrtxn_t* txn) {
+  if (!nr_txn_log_forwarding_enabled(txn)) {
+    return false;
+  }
+
+  if (!txn->options.log_forwarding_context_data_enabled) {
+    return false;
+  }
+
+  return true;
```

Comment:
Could simplify to this:
```suggestion
  return txn->options.log_forwarding_context_data_enabled
```

Reply:
I actually prefer the current form where a sequence of filters must pass before returning `true`.  The compiler will generate the code you have but for reading and modifying I prefer the expanded form.